paasta_tools/mesos_tools.py

```diff
@@ -47,6 +49,7 @@
 
 
 DEFAULT_MESOS_CLI_CONFIG_LOCATION = "/nail/etc/mesos-cli.json"
+DEFAULT_SERVICES_LOCATION = "/nail/etc/services/services.yaml"
```

Comment:
I mean, this adds a dependency on paasta. If someone wants to use smartstack in taskproc, then they have to add paasta as a dependency. Also, paasta has its own utils to figure out master address. Does it make sense to keep this in paasta?